stubs/SQLAlchemy/sqlalchemy/dbapi.pyi

```diff
@@ -0,0 +1,36 @@
+# TODO: Tempory copy of _typeshed.dbapi, until that file is available in all typecheckers.
```

Comment:
It might be good to add a comment to `_typeshed.dbapi`, to make sure these stay in sync, or even test it with `tests/check_consistent.py`.

Reply:
I would prefer to just get rid of this copy ASAP. (Probably when the next major mypy version is released, which hopefully should be faster now.)